lib/tilt/jbuilder.rb

```diff
@@ -103,9 +103,10 @@ def evaluate(scope, locals, &block)
     private
     def set_locals(locals, scope, context)
       view_path = options[:view_path]
-      scope.send(:instance_variable_set, '@_jbuilder_view_path', view_path)
-      scope.send(:instance_variable_set, '@_jbuilder_locals', locals)
-      scope.send(:instance_variable_set, '@_tilt_data', data)
+      scope.instance_exec(binding) {
+        @_jbuilder_view_path = view_path
+        @_jbuilder_locals = locals
+      }
```

Comment:
Semi-related: I removed the line that was setting `data` because it wasn't clear to me where `data` was previously instantiated, and because the tests were still passing without it.